test/conftest.py

```diff
@@ -152,7 +151,8 @@ def __init__(self, blob_host, blobs):
         self.volume_reloads: Dict[str, int] = defaultdict(lambda: 0)
 
         self.sandbox_defs = []
-        self.sandbox: subprocess.Popen = None
+        self.sandbox: asyncio.subprocess.Process = None
+        self.is_shell = False
```

Comment:
`self` is here the Servicer, so this naming suggests that the variable is concerned with whether _the Servicer_ is a shell or not. That obviously makes no sense, so it should be something like `self.sandbox_is_shell` 🙂